### ISSUE TITLE
Aliased function with keyword-only arguments produces unknown type error

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#490]
**Describe the bug**
When aliasing a function with a bare `*` an unknown variable type error is emitted.  The error message does not point to which part of the definition is unknown.  It appears as though Pyright expects a bare `*` to be typed though that might very well be a symptom and not the cause.

**To Reproduce**
Type check the snippet below.

**Expected behavior**
That the type of `bar` would be known.

**Screenshots or Code**
```py
# pyright: strict

def foo(*, value: str):
    return None

bar = foo
```

**VS Code extension or command-line**
extension, 1.1.21

**Comment by erictraut** [comment]
Thanks for the bug report. Yes, this is a bug, and it's easy to fix. It will be addressed in the next version.

**Comment by erictraut** [comment]
This is fixed in Pyright 1.1.22, which I just published.
